compiler/x/codegen/OMRTreeEvaluator.cpp

```diff
@@ -4779,6 +4779,152 @@ TR::Register *OMR::X86::TreeEvaluator::atomicorEvaluator(TR::Node *node, TR::Cod
    return NULL;
    }
 
+TR::Register *                                                                                                                                                             
+OMR::X86::TreeEvaluator::tstartEvaluator(TR::Node *node, TR::CodeGenerator *cg)                                                                                             
+   {                                                                                                                                                                       
+   /*         
+   .Lstart:
+      xbegin       .Lfallback
+      jmp          .LfallThrough
```

Comment:
changing it would presumably affect things other than JitBuilder though, so let's leave it for now.